ui/components/multichain/edit-networks-modal/edit-networks-modal.js

```diff
@@ -0,0 +1,83 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { TextVariant } from '../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { getNonTestNetworks, getTestNetworks } from '../../../selectors/index';
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  Checkbox,
+  Text,
+  Box,
+} from '../../component-library';
+import { NetworkListItem } from '../index';
+
+export const EditNetworksModal = ({ onClose }) => {
+  const t = useI18nContext();
+  const nonTestNetworks = useSelector(getNonTestNetworks);
+  const testNetworks = useSelector(getTestNetworks);
+  return (
+    <Modal
+      isOpen
+      onClose={() => {
+        onClose();
+      }}
+      className="import-nfts-modal"
+    >
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader
+          onClose={() => {
+            onClose();
+          }}
+        >
+          {t('editNetworksTitle')}
+        </ModalHeader>
+        <Box padding={4}>
+          <Checkbox
+            label={t('selectAll')}
+            isChecked={true}
+            // onClick={() => (allAreSelected() ? deselectAll() : selectAll())}
```

Comment:
would be nice to have this click handler as part of this PR, unit testing select all with mock props/local state would then be possible

Reply:
This PR is just to get the skeleton ready. I will add more tests when we will make it functional